src/index.js

```diff
@@ -29,13 +29,13 @@ export const MeetingOrganizer = ({ outsideJwt, meetingOrganizerBaseLink, userDat
       })
     } else {
       api.register(credentials)
-      .then(_ => setView(loginView))
+        .then(_ => setView(loginView))
```

Comment:
Tak w sumie nie powinniśmy sprawdzać co dostaliśmy z rejestraci?

Reply:
Rejestracja może się jakoś wywalić?

Reply:
Wait, to jest jak najbardziej legitne. W większości fetchy używam Twojej nakładki fetchData i ona tam ma zakodowane przechwytywanie błędów. Więc jeśli tu jesteśmy, to znaczy, że wszystko poszło dobrze, więc jest git.